clm_demeter/prepare_baselayer.py

```diff
@@ -0,0 +1,161 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt # to generate plots
+from netCDF4 import Dataset     # to work with NetCDF files
```

Comment:
These need to be added into our `requirements.txt` file if they are not present so they are installed when the user installs the Python package we are building.

Reply:
Also, `netCDF4` requires a C dependency that may complicate things a bit but we will test this.

Reply:
I don't need to import matplotlib.pyplot. I used it to temporarily check the netcdf variables read and whether flipud worked correctly. I will remove the line in the next round of edits.